Dockerfile.eol-checker

```
@@ -0,0 +1,29 @@
+
+LABEL summary="${SUMMARY}" \
+    description="${DESCRIPTION}" \
+    io.k8s.description="${DESCRIPTION}" \
+    io.k8s.display-name="EOL Checker for SCL org projects" \
+    io.k8s.vendor="SCL org" \
+    architecture="x86_64"
+
+RUN dnf install -y python3.13-pip git && \
+    dnf clean all
+
+COPY requirements.sh requirements.txt "${WORK_DIR}"
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)